packages/cfd/src/Containers/props.types.ts

```diff
@@ -328,7 +327,6 @@ export type TCFDDbviOnboardingProps = {
     fetchAccountSettings: () => void;
     has_created_account_for_selected_jurisdiction: boolean;
     has_submitted_cfd_personal_details: boolean;
```

Comment:
i think we can remove this "has_submitted_cfd_personal_details". can you check this